### DISCUSSION TITLE
Habitat

### DISCUSSION THREAD

**Discussion by yagiz-dev** [iptv-org/community#725] [Q&A]
**Channel Name:** Habitat TV
**Channel ID:** Habitat.tr
**Country:** tr
**Language:** tr
**Website:** http://habitattv.com.tr/ (only accessible over HTTP)
**Stream URL (optional):** xxx
**Notes (optional):** xxx

**Comment by Sphinxroot** [comment]
https://www.canlitv.mobi/habitat-tv
https://canlitvizle.com/habitat-tv
https://www.canlitv.vin/habitat-tv
https://www.canlitv.dev/habitat-tv
https://www.canlitvizle.plus/habitat-tv

all have token

**Comment by BellezaEmporium** [comment]
> 
> 
> https://www.canlitv.mobi/habitat-tv https://canlitvizle.com/habitat-tv https://www.canlitv.vin/habitat-tv https://www.canlitv.dev/habitat-tv https://www.canlitvizle.plus/habitat-tv
> 
> all have token

Weird scripting involved behind this.

**Comment by UzunMuhalefet** [comment]
https://59c7ea6bec1c6.streamlock.net/filbox/habibattv.stream/playlist.m3u8

**Comment by Dum4G** [comment]
http://istanbul.tvizlehd.com/live/habitattv.m3u8
https://ankara.tvizlehd.com/live/habitattv.m3u8
https://59c7ea6bec1c6.streamlock.net/live/habibattv.stream/playlist.m3u8

Are those working ? @evrifaessa

**Comment by BellezaEmporium** [comment]
Upon testing, the first 2 work, the last one is a 403.
We shall precise that those are 2 different streams.

One shall be Habitat Istanbul and the other one Habitat Ankara.

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19744
